.scripts/ci.yml

```diff
@@ -263,6 +263,12 @@ stages:
           - script: npm run test
             workingDirectory: $(Build.SourcesDirectory)/packages/cadl-rlc-emitter
             displayName: "Generate RLC Test Clients from Cadl-ranch"
+          - script: npm run coverage
+            displayName: "Check Coverage"
+            workingDirectory: $(Build.SourcesDirectory)/packages/cadl-rlc-emitter
+            continueOnError: true
+          - publish: $(Build.SourcesDirectory)/packages/cadl-rlc-emitter/cadl-ranch-coverage-typescript.json
+            artifact: CoverageReport
```

Comment:
Can you add a condition like https://github.com/Azure/autorest.typescript/blob/main/.scripts/ci.yml#L182 to make sure the test coverage will only get upload when we are running the internal pipeline.